### PR TITLE
Protobuf infrastructure

### PR DESCRIPTION
**Pull request by mccorby**

### CODE REVIEW

File: syftlib/src/test/resources/proto_files/plan.json [hunk 1]
```diff
@@ -0,0 +1,1554 @@
...
+      "ownerPlaceholder": {
+        "id": {
+          "idInt": "63605762519"
+        },
+        "tags": [
+          "#input-4",
+          "#1"
+        ]
+      },
+      "returnPlaceholders": [
```
Comment (vkkhare) [inline syftlib/src/test/resources/proto_files/plan.json]: Do we need to worry about individual actions? I thought we would be getting a torchscript directly

File: syftlib/src/main/java/org/openmined/syft/framework/PlanExecutor.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package org.openmined.syft.framework
+
+import android.content.Context
+import com.google.protobuf.ByteString
+import org.pytorch.Module
+import java.io.File
+import java.io.FileOutputStream
+
+class PlanExecutor {
+
```
Comment (vkkhare) [inline syftlib/src/main/java/org/openmined/syft/framework/PlanExecutor.kt]: I made a plan class for our purposes in the previous PR but am not sure whether we would need to override the syft-proto plan class with our custom execute implementations

File: syftlib/src/test/resources/proto_files/plan.json (hunk 1 above)
Comment (mccorby) [reply syftlib/src/test/resources/proto_files/plan.json]: Yes, this should go. I was using it when trying to understand what was going on.

File: syftlib/src/main/java/org/openmined/syft/framework/PlanExecutor.kt (hunk 2 above)
Comment (mccorby) [reply syftlib/src/main/java/org/openmined/syft/framework/PlanExecutor.kt]: I need to think about this. It was still part of the draft to put things together and see that the deserialization was happening. I'll be back to it and consider your alternative.

File: syftlib/src/main/java/org/openmined/syft/Syft.kt [hunk 3]
```diff
@@ -56,6 +56,7 @@ class Syft private constructor(
                 }
                 .subscribeOn(schedulers.computeThreadScheduler)
                 .observeOn(schedulers.calleeThreadScheduler)
+                .onErrorReturn{ print("Something went wrong $it") }
```
Comment (vkkhare) [inline syftlib/src/main/java/org/openmined/syft/Syft.kt]: this is removed in the new pattern ;p

File: syftlib/src/main/java/org/openmined/syft/networking/clients/MessageProcessor.kt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package org.openmined.syft.networking.clients
+
+import org.openmined.syftproto.types.torch.v1.ScriptModuleOuterClass
+
+class MessageProcessor {
```
Comment (vkkhare) [inline syftlib/src/main/java/org/openmined/syft/networking/clients/MessageProcessor.kt]: Will this be going to `jobstatusprocessor` now?

Comment (mccorby) [reply syftlib/src/main/java/org/openmined/syft/networking/clients/MessageProcessor.kt]: It should be invoked from there, yes but it should be its own entity.

File: syftlib/src/main/java/org/openmined/syft/Syft.kt (hunk 3 above)
Comment (mccorby) [reply syftlib/src/main/java/org/openmined/syft/Syft.kt]: We can remove it after merging the other PR. This line fixes a crash.

**Comment by vkkhare** [comment]
Rebase needed

File: syftlib/src/main/java/org/openmined/syft/Classifier.kt [hunk 5]
```diff
@@ -0,0 +1,75 @@
+package org.openmined.syft
+
+import android.graphics.Bitmap
+import org.pytorch.IValue
+import org.pytorch.Module
+import org.pytorch.Tensor
+import org.pytorch.torchvision.TensorImageUtils
+import java.io.File
+
+class Classifier (forwardPath: String,backwardPath:String,x:String,y:String){
```
Comment (vkkhare) [inline syftlib/src/main/java/org/openmined/syft/Classifier.kt]: This shouldn't be in the syftlib

**Comment by mccorby** [comment]
About the duplicated files, they are actually different. One is used only to test that the deserialization of any model will work.
We could use the one in the demo app but then we would need to change the resources location for all tests.
Also note that files in tests are not included in the APK.

**Comment by vkkhare** [comment]
Cool In that case am merging the PR

**Pull request merged**